scripts/generate_index_files.py

```diff
@@ -105,10 +111,16 @@ def curate_tad_elements(tad_df, input_df, gen_class):
                                     ignore_index=True)
             bnd_df = bnd_df.append(o_df, ignore_index=True)
 
-        # Assign TAD information
-        elem_sub_df['TAD_id'] = tad_id
-        elem_sub_df['TAD_start'] = start
-        elem_sub_df['TAD_end'] = end
+        if gen_class == 'LD':
```

Comment:
These scripts should be split into modules with the actual calls being doone in notebooks as evidenced by this special casing and the hardcoding above:

```python
 if TAD_CELL in ['hESC', 'IMR90']:
     BASE_DIR = 'data/hg/'
     GENOME = 'hg19'
     REF_GENE_GTF = BASE_DIR + 'gencode.v19.annotation.gtf.gz'
     TAD_LOC = BASE_DIR + TAD_CELL + '_domains_hg19.bed'
     REF_SNPS = BASE_DIR + 'hg_common-snps.tsv'
 elif TAD_CELL in ['mESC', 'cortex']:
     BASE_DIR = 'data/mm/'
     GENOME = 'mm9'
     REF_GENE_GTF = BASE_DIR + 'gencode.vM9.annotation.gtf.gz'
     TAD_LOC = BASE_DIR + TAD_CELL + '_domains_mm9.bed'
     REF_SNPS = BASE_DIR + 'mm_common-snps.tsv'
 else:
     raise ValueError('Please input either "hESC", "IMR90", "mESC", or '
```

Outside of scope of this PR. Something to keep in mind for the future.